core/src/main/java/org/apache/brooklyn/core/entity/internal/EntityConfigMap.java

```diff
@@ -132,7 +132,11 @@ public EntityConfigMap(AbstractEntity entity, Map<ConfigKey<?>, Object> storage)
 
         // Get own value
         if (((ConfigKeySelfExtracting<T>)key).isSet(ownConfig)) {
-            ownValue = Maybe.of(((ConfigKeySelfExtracting<T>)key).extractValue(ownConfig, exec));
+            Map<ConfigKey<?>, ?> ownCopy = null;
```

Comment:
But the compiler won't compile if you don't? Or is that just my misleading Eclipse settings...?